### DISCUSSION TITLE
User-voted compatibility flags (for Minor Minecraft patches)

### DISCUSSION THREAD

**Discussion by ElectroArtifex** [modrinth/.github#19] [Feature requests]
### Is your feature request related to a problem? Please describe.

Discerning the compatibility of a 1.x.1-flagged mod with a 1.x.2 update is always tedious and a chore unless the dev has explicitly stated its compatibility. Because so many people use mod launchers these days, it means many mods that would work fine go unseen until the dev manually changes the flag. This can also lead to users nagging devs to update the flags in all the places where the mod is hosted. 

### Describe the solution you'd like

I suggest allowing logged-in users on Modrinth the ability to "vote" whether a mod flagged with a major-release version is compatible with the minor release update in their experience, and then giving a checkbox in the version search options for "soft version filter" to include 1.x.1 mods with 90%+ voted compatibility rating in their 1.x.2 search or otherwise "hard version filter" for only the releases the mod dev has flagged as compatible. To go a step further, it could even prompt mod devs with a notification of "your mod [Example Mod] has been voted compatible with 1.x.2. Approve?" for an easy update. 

This could also pave the way in future for a system where inter-mod compatibility can be rated with mods commonly downloaded by the same users, making the task of modpack creation a little easier as well. 

### Additional context

I realize implementing this would take considerable work, but I truly believe this could be a system that would help all categories of the Modrinth userbase in time, and remove much of the frustration around modding or using mods for incremental minecraft patches.

**Comment by ryanccn** [comment]
tbf this is a great idea. I still have about a dozen mods that haven't marked themselves as 1.19.2 compatible, but 1.19 works perfectly well. in these cases it would be useful for users to be able to mark it as compatible for other people

**Comment by MopsTMC** [comment]
I use ferium and this is kind of a pain to deal with because you have to provide file overrides for at least 10 mods, so this could be an awesome addition

**Comment by intergrav** [comment]
Agreed

**Comment by Wuzado** [comment]
Also related - *a lot* of plugins depend on relatively stable Spigot/Paper API's. Thus, it's not uncommon to have an old, abandoned plugin working across many major (1.x) versions, even when they have a totally distant, old native Minecraft version.
